.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +16,7 @@ PR submission checklist:
 - [ ] I have completed the Regression Notes.
 - [ ] I have considered adding unit tests for my changes.
 - [ ] I have considered adding accessibility improvements for my changes.
+- [ ] I have tested this feature for both WordPress.com sites and self-hosted Jetpack sites.
```

Comment:
It seems a bit too specific for the PR submission checklist.
Would you consider adding it under the "Testing checklist" (renamed from "UI Changes testing checklist" below?

Reply:
Done! [fa386ec](https://github.com/wordpress-mobile/WordPress-iOS/pull/22493/commits/fa386ec9ed6dff8907f1e3bd6c385f106c108df2)